JS

```diff
@@ -0,0 +1,211 @@
+  name: 'Cola',
+  price: 50,
+  calories: 40
+};
+
+SaladAndDrink.prototype.DRINK_COFFEE = {
+  name: 'Coffee',
+  price: 80,
+  calories: 20
+};
+
+       //// different functions for a big and a small because it is more conviniently 
+let arrOrder = [];
+   
+function addBigHamburger(){
+  let count = document.getElementById('bigHamburgerCount').value;
+  let stuffing = document.getElementById('bigHamburgeStuffing').value;
+  let order = new Hamburger('LARGE_SIZE', count, stuffing);
+  if(count!=0) addCurrent(order);
+  if(!count) alert('Please enter the number');
+     //// fill the table
+  orderList();
+   /// count totall summ and calories
+   totall();
+}
+    
+function addSmallHamburger(){
+  let count = document.getElementById('smallHamburgerCount').value;
+  let stuffing = document.getElementById('smallHamburgerStuffing').value;
+  let order = new Hamburger('SMALL_SIZE', count, stuffing);
+  if(count !=0) addCurrent(order);
+  if(!count) alert('Please enter the number')
+  orderList();
+  totall();      
+}
+
+function addSalad(){
+  let type  = document.getElementById('chooseSalad').value;
+  let count = document.getElementById('saladCount').value;
+  let order = new SaladAndDrink(type, count);
+  if(count !=0) addCurrent(order);
+  if(!count) alert('Please enter the number')
+  orderList();
+  totall();
+}
+
+function addDrink(){
+  let type= document.getElementById('chooseDrink').value;
+  let count = document.getElementById('drinkCount').value;
+  let order = new SaladAndDrink(type, count);
+  if(count !=0) addCurrent(order);
+  if(!count) alert('Please enter the number');
+  orderList();
+  totall();
+}
+
+function orderList(){
+ var tmpl = document.getElementById('grid-template').innerHTML.trim();
+ tmpl = _.template(tmpl);
+
+
+ document.getElementById('grid-holder').innerHTML = tmpl({
+   list: arrOrder
+ }); 
+}
+ 
+
+function totall() {
+  let summ =0;
+  let energy =0;
+  for (let i=0; i<arrOrder.length; i++) {
+    summ = summ + arrOrder[i].calculatePrice();
+    energy = energy + arrOrder[i].calculateCalories();
+  }
+  document.getElementById('totallSumm').innerHTML = summ;
+  document.getElementById('totallCalories').innerHTML = energy;
+}
+
+function deleteCurrentItem() {
+  $('td').on('click', function () {
+    var $td = $(this),
+    $tr = $td.parent();
+    arrOrder.splice($tr.index(),1);
+    orderList();
+    totall();
+  });
+}
+
+function addCurrent(order) {
+  arrOrder.push(order);
+  let t = arrOrder.length-1;
+  if (t ==0) { return;}
+  for (let i= t-1; i>=0; i--){
+    if (arrOrder[i].getName()==arrOrder[t].getName()) {
+      arrOrder[i].count = (+arrOrder[i].count)+(+arrOrder[t].count);
+      arrOrder.pop();
+      return;
+    }
+  }
+}
+
+function openOrder (){
+  arrOrder = [];
+  orderList();
+  totall();
+  document.getElementById("hasPaid").innerHTML = " ";
+  let inputNumber =  document
+  .querySelectorAll("input[type='number']");
+  for (let element of inputNumber){
+    element.value=0;
```

Comment:
@Mikleprog after that operation user will not see an alert to type any number in input